src/core/egg-herbie.rkt

```diff
@@ -0,0 +1,247 @@
+#lang racket
```

Comment:
This makes so much more sense.

Reply:
Hilariously, I was thinking of doing this on my branch as well.

Reply:
I think we should plan to do:
- Submit everything going to PLDI messing with Egg
- Merge this branch
- Figure out how to fix your branch so it's a clean merge

Reply:
Yep. There's quite a bit more overlap. I made nearly the same interface changes for functions like `egraph-get-variants` and `egraph-get-simplest`.